src/librustc_mir/transform/nll.rs

```diff
@@ -143,4 +147,9 @@ impl MirPass for NLL {
             let _results = visitor.into_results();
         })
     }
-}
\ No newline at end of file
+}
+
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
```

Comment:
We couldn't keep the derive for `Hash` because of `FxHashSet`. It errors out saying that `HashSet` itself doesn't implement `Hash`.

Reply:
Hopefully this will not have to be `Hash`.